code/controllers/subsystem/ore_generation.dm

```diff
@@ -58,6 +37,20 @@ SUBSYSTEM_DEF(ore_generation)
 				if(stallbreaker >= length(possible_vents))
 					return SS_INIT_SUCCESS //We've done all we can here.
```

Comment:
We don't want to early return else we will skip all logging. Instead just break the loop like this
```suggestion
				if(stallbreaker >= length(possible_vents))
					break //We've done all we can here.
				continue
		if(stallbreaker >= length(possible_vents))
			break //We've done all we can here.
```
One to break the inner loop & next to break the outer loop